app/views/unattended/provisioning_templates/provision/preseed_default.erb

```diff
@@ -114,6 +114,7 @@ d-i apt-setup/local<%= repos %>/repository string <%= medium[:url] %> <%= @host.
 d-i apt-setup/local<%= repos %>/repository string http://debian.saltstack.com/debian <%= @host.operatingsystem.release_name %>-saltstack main
 d-i apt-setup/local<%= repos %>/comment string SaltStack Repository
 d-i apt-setup/local<%= repos %>/key string http://debian.saltstack.com/debian-salt-team-joehealy.gpg.key
+d-i apt-setup/services-select multiselect
```

Comment:
Would this be better?
```suggestion
<% if plugin_present?('katello') %>
  d-i apt-setup/services-select multiselect
<% end %>
```

Reply:
I'm not sure. Just because you installed Katello, does that mean that mirrored content is used? I don't think it is.

Reply:
So I am making it depend on the parameter `kt_activation_keys` for now.
If that parameter does not meet the requirements, then the only thing I can think of is introducing a new parameter. Perhaps something like `katello_content_only`?